### DISCUSSION TITLE
Vessel count in title

### DISCUSSION THREAD

**Discussion by flygfantast** [jvde-github/AIS-catcher#260] [Ideas]
It would be nice to see the vessel count in the (tab) title, like VRS or SkyAware.

Could be a toggle on/off feature.

![image](https://github.com/jvde-github/AIS-catcher/assets/124347186/f8a72dca-e6d5-449b-ab74-3beb98e990ab)

**Comment by jvde-github** [comment]
Nice idea

**Comment by jvde-github** [comment]
![Screenshot from 2024-04-27 12-44-05](https://github.com/jvde-github/AIS-catcher/assets/52420030/af460a3b-2d04-432d-83fa-6d406f3e9dc3)

First version there.... Vessel count is updated once in the map tab.

**Comment by jvde-github** [comment]
Moved the counter to the front though as it will typically drop off

**Comment by flygfantast** [comment]
Nice, thanks! :)
